extensions/email/src/main/java/org/mule/extension/email/internal/AbstractEmailConnectionProvider.java

```diff
@@ -22,43 +24,46 @@
 {
 
     /**
-     * The socket connection timeout. This attribute
-     * works in tandem with {@link #timeoutUnit}.
+     * The socket connection timeout value. This attribute works in tandem with {@link #timeoutUnit}.
      * <p>
      * Defaults to {@code 5}
      */
     @Parameter
     @Optional(defaultValue = "5")
+    @Placement(tab = "Advance", group = "Timeout Configuration", order = 2)
+    @Summary("Socket connection timeout value")
     private int connectionTimeout;
 
     /**
-     * The socket read timeout value. This attribute
-     * works in tandem with {@link #timeoutUnit}.
+     * The socket read timeout value. This attribute works in tandem with {@link #timeoutUnit}.
      * <p>
      * Defaults to {@code 5}
      */
     @Parameter
     @Optional(defaultValue = "5")
+    @Placement(tab = "Advance", group = "Timeout Configuration", order = 3)
```

Comment:
Define constant for TimeoutConfiguration. This one probably makes sense to do at the extension level instead of the annotation